lib/templates.js

```diff
@@ -158,8 +162,8 @@ const createAPIResourceTemplate = (serviceName, serviceDefinition, appConfig, pa
                 DESCRIPTION_PREFIX + serviceName,
             version: "1.0.0",
             lastUpdate: appConfig.lastUpdate,
-            visibility: "public",
-            partOfPackage: _getPackageID(appConfig.ordNamespace, packageIds, "api"),
+            visibility: RESOURCE_VISIBILITY.public, // default visibility
```

Comment:
The constant is great! However, the comment behind should be removed, no needed I guess?